macros/g4simulations/G4_CEmc_EIC.C

```diff
@@ -269,6 +269,11 @@ void CEMC_Clusters(int verbosity = 0) {
     cemc_clusterbuilder->Detector("CEMC");
     cemc_clusterbuilder->Verbosity(verbosity);
 
+    cemc_clusterbuilder->set_threshold_energy(0.030); // This threshold should be the same as in CEMCprof_Thresh**.root file below
+    std::string femc_prof = getenv("CALIBRATIONROOT");
+    femc_prof += "/EmcProfile/CEMCprof_Thresh30MeV.root";
+    cemc_clusterbuilder->LoadProfile(femc_prof.c_str());
```

Comment:
Hi Sasha

`cemc_clusterbuilder` is declared as `RawClusterBuilderGraph` and do not have `LoadProfile` function. Would you like to switch it to your `RawClusterBuilderTemplate`?